src/main/java/frc/robot/commands/Arm/ScoreInSpeakerHigh.java

```diff
@@ -24,6 +24,7 @@ public ScoreInSpeakerHigh(PizzaBoxSubsystem pizzaBoxSubsystem, ArmSubsystem armS
         new StopNoteContainerMotor(pizzaBoxSubsystem),
         new SpinToArmAngle(armSubsystem, ArmSubsystem.Arm.INTAKE_ANGLE)
     );
+    pizzaBoxSubsystem.hasNote = false;
```

Comment:
Should this be in initialize, or the first 'InstantCommand' so it gets set false every time?